src/webgpu/api/validation/render_pass/resolve.spec.ts

```diff
@@ -0,0 +1,120 @@
+export const description = `API Validation Tests for RenderPass Resolve.
+
+  Test Coverage:
+    - When resolveTarget is not null:
+      - Test that the colorAttachment is multisampled:
+        - A single sampled colorAttachment should generate an error.
+      - Test that the resolveTarget is multisampled:
+        - A multisampled resolveTarget should generate an error.
+      - Test that the resolveTarget has usage OUTPUT_ATTACHMENT:
+        - A resolveTarget without usage OUTPUT_ATTACHMENT should generate an error.
+      - Test that the resolveTarget's texture view describes a single subresource:
+        - An error should be generated when:
+          - The resolveTarget view mip count is not 1.
```

Comment:
Also test with the baseArrayLayer and baseMipLevel !=0 ?

Reply:
Done.

Reply:
@Kangz did you mean {count=1, base>0} or {count>1, base>0}?
Latter is now in this PR; former could (maybe should) be tested here but will need to be included in operation tests too.

Reply:
Yeah I meant (count = 1, base > 0)

Reply:
Added to the description.